css/index.css

```diff
@@ -69,43 +69,73 @@ h1, h2, h3, h4, h5 {
 :root{
 	font-size: 16px;
 }
+@media(max-width:500px){
+	:root{
+		font-size: 24px;
+	}
+}
 
 .site{
-	
-	align-items: center;
 	display: flex;
 	flex-direction: column;
+	align-items: center;
+}
 
+@media(max-width:800px){
+	.site{
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+	}
```

Comment:
the .site is already flex so you don't have to put it again, it makes your code mode DRY